bin/stacks/routing-database-stack.ts

```diff
@@ -5,8 +5,22 @@ import { Construct } from 'constructs'
 
 export interface RoutingDatabaseStackProps extends cdk.NestedStackProps {}
 
+export const V3PoolsDynamoDbTableName = 'V3PoolsCachingDB'
```

Comment:
For consistency, could we do the same in the CachedRoutes Table?

Reply:
Yep. I did logical grouping of DynamoDB constants and refactored cached routes table attributes accordingly.